pyproject.toml

```diff
@@ -29,6 +29,7 @@ classifiers = [
 python = ">=3.7,<4.0"
 Django = ">=3.2"
 strawberry-graphql = ">=0.154.0"
+django-debug-toolbar = { version = ">=3.4", optional = true }
```

Comment:
Probably also need to add a `tool.poetry.extras` section

Reply:
done !